packages/merge-styles/src/keyframes.ts

```diff
@@ -0,0 +1,42 @@
+import { getVendorSettings } from './getVendorSettings';
+import { Stylesheet } from './Stylesheet';
+import { serializeRuleEntries } from './styleToClassName';
+
+/**
+ * Registers keyframe definitions.
+ *
+ * @public
+ */
+export function keyframes(timeline: { [key: string]: {} }): string {
+  let vendorSettings = getVendorSettings();
+  let stylesheet = Stylesheet.getInstance();
+  let name = stylesheet.getClassName();
+
+  let rulesArray: string[] = [];
+
+  for (let prop in timeline) {
+    if (timeline.hasOwnProperty(prop)) {
+      rulesArray.push(prop, ' {', serializeRuleEntries(timeline[prop]), '}');
+    }
+  }
+  let rules = rulesArray.join('');
+
```

Comment:
But i've removed the prefixes because ie11 and chrome and firefox and safari and opera all support keyframes.